src/Compute/Compute/Generated/Models/PSDiskUpdate.cs

```diff
@@ -43,6 +43,8 @@ public partial class PSDiskUpdate
 
         public string Tier { get; set; }
         public bool? BurstingEnabled { get; set; }
+        public PurchasePlan PurchasePlan { get; set; }
```

Comment:
I prefer PSPurchasePlan as the type of PurchasePlan so that we can use the return object of `New-AzDiskPurchasePlanConfig` to set PSDiskUpdate's property PurchasePlan directly

Reply:
Changed to PSPurchasePlan